frontend/src/metadata/popups/NewSlicePopup.svelte

```diff
@@ -82,13 +84,13 @@
 
 		// if slices are not empty in $selections
 		if ($selections.slices.length > 0) {
-			let slicesPredicates: FilterPredicateGroup = { predicates: [], join: "" };
+			let slicesPredicates;
 			$selections.slices.forEach((s, i) => {
 				let sli_preds = $slices.get(s).filterPredicates;
 				if (i !== 0) {
 					sli_preds.join = "&";
 				}
-				slicesPredicates.predicates.push(sli_preds);
```

Comment:
The type should be predicate instead of predicate group when creating a derived(nested) slice from the current selected slice.